recipes/server_install.rb

```diff
@@ -265,6 +282,3 @@
     end
   end
 end
-
-include_recipe 'rundeck::_connect_rundeck_api_client'
```

Comment:
I think we shloud keep the server_install recipie just to install the rundeck server. I am not using the project creation from cookbooks. So moved those parts to server recipie.